### PR TITLE
Update korean readme

### PR DESCRIPTION
**Pull request by dsdanielpark**
As a Korean, I have naturally corrected the Korean translation version of Chat GPT.

### CODE REVIEW

File: readme/README_ko.md [hunk 1]
```diff
@@ -336,26 +342,26 @@ cd -
...
 ## Citation
-이 repository를 유용하게 사용하셨다면 ⭐을 눌러주시고 cite해주시기 바랍니다:
+이 repository를 유용하게 사용하셨다면 ⭐을 눌러주시고 다음을 통해 인용해주시면 감사하겠습니다.
 
-```
+```bibtex
 @misc{lmflow,
   author = {Shizhe Diao and Rui Pan and Hanze Dong and KaShun Shum and Jipeng Zhang and Wei Xiong and Tong Zhang},
-  title = {LMFlow: An Extensible Toolkit for Finetuning and Inference of Large Foundation Models},
+  title = {LMFlow: An Extensible Toolkit for 파인튜닝 and Inference of Large Foundation Models},
```
Comment (shizhediao) [inline readme/README_ko.md:364]: I'd prefer to keep the title unchanged.

**Comment by shizhediao** [comment]
Hi,
Thank you so much for your help!
I think we can merge it soon after some minor modifications. (keep the title unchanged)
Let me know if you have any questions~

File: readme/README_ko.md (hunk 1 above)
Comment (dsdanielpark) [reply readme/README_ko.md]: I have reviewed the content and made corrections to the entire text. I would like to contribute to the Korean documentation by translating and optimizing the code for future updates. I am impressed by this amazing project.

**Comment by dsdanielpark** [comment]
I have reviewed the content and made corrections to the entire text.

**Review by shizhediao — APPROVED** [review]
Thank you so much for your contribution! 
The Korean README looks better now with your help.

**Pull request merged**